lib-injection/host_inject.rb

```diff
@@ -23,30 +23,26 @@ def debug_log(msg)
   require "shellwords"
   require "fileutils"
 
-  failure_prefix = "Datadog lib injection failed:"
   support_message = "For help solving this issue, please contact Datadog support at https://docs.datadoghq.com/help/."
 
   unless Bundler::SharedHelpers.in_bundle?
-    debug_log "[ddtrace] Not in bundle... skipping host injection"
+    debug_log "[ddtrace] Not in bundle... skipping injection"
     return
   end
```

Comment:
Since a lot of stuff between both files is copy/pasted, consider adding a note atop both files mentioning that and reminding people to keep them in sync.